### PR TITLE
Restructure the documents

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@xuechunL PTAL.

File: README.md [hunk 1]
```diff
@@ -23,19 +21,20 @@
...
-  - [Performance Tuning](op-guide/tune-TiKV.md)
-  + Backup and Restore
-    - [Reading Data from History Versions](op-guide/history-read.md) 
-  - [Import and Export](op-guide/migration.md)
+  + Performance Tuning
+    - [Performance Tuning for TiKV](op-guide/tune-TiKV.md)
+  + Backup and Migrate
+    + [Migrate data from MySQL to TiDB](op-guide/migration.md)
+      - [Migrate all the data](using-the-mydumper--loader-tool-to-export-and-import-all-the-data)
+      - [Migrate the data incrementally](optional-using-the-syncer-tool-to-import-data-incrementally)
```
Comment (andelf) [inline README.md]: above 2 links are not exists

File: README.md [hunk 2]
```diff
@@ -52,10 +51,17 @@
     - [Miscellaneous Functions](sql/miscellaneous-functions.md)
   - [TiDB SQL Grammar](sql/README.md)
   - [Compatibility with MySQL](sql/mysql-compatibility.md)
++ Advanced usage
```
Comment (andelf) [inline README.md]: Title Case?

File: op-guide/migration.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ Before using the `syncer` tool, make sure:
     ```
 
 
-## Step 1. Using the `checker` tool to check the Schema
+## 1. Using the `checker` tool to check the Schema
```
Comment (andelf) [inline op-guide/migration.md]: better remove all digit title prefix

**Comment by andelf** [comment]
LGTM

**Pull request merged**